helm/superset/templates/deployment-worker.yaml

```diff
@@ -28,7 +28,11 @@ metadata:
     {{ toYaml .Values.supersetWorker.deploymentAnnotations | nindent 4 }}
 {{- end }}
 spec:
+  {{- if .Values.supersetWorker.replicaCount }}
+  replicas: {{ .Values.supersetWorker.replicaCount }}
+  {{- else }}
```

Comment:
Just make a new value under `supersetWorker` and use it here (default of the current `replicaCount`)

Reply:
Ok so I did
- remove `replicaCount` in the values, in the schema, and in the array of required values
- add `supersetNode.replicaCount` and `supersetWorker.replicaCount` in the values, in the schema, and in the array of required values
- edit the "Running on Kubernetes" doc